### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by edcarSky** [ZenRows/youtube-video-viral#1]
error on console please help

DevTools failed to load source map: Could not load content for https://cdn.jsdelivr.net/npm/chartjs-adapter-luxon@1.0.0/dist/chartjs-adapter-luxon.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by AnderRV** [comment]
Is the error preventing your from loading the graphs? They should display anyway.
Maps are a debugging technique used in browser with minified JS and CSS files, chartjs-adapter-luxon in this case. It looks like the include the sourceMaps in their code, but then jsdelivr does not have that file.
`//# sourceMappingURL=chartjs-adapter-luxon.min.js.map`

If the problem remains, we could change the minified version in the HTML for the un-minified one: `https://cdn.jsdelivr.net/npm/chartjs-adapter-luxon@1.0.0/dist/chartjs-adapter-luxon.js`

Let us know you cannot access the content or there is some other error. Thanks!

**Comment by edcarSky** [comment]
If the problem remains, we could change the minified version in the HTML for the un-minified one: https://cdn.jsdelivr.net/npm/chartjs-adapter-luxon@1.0.0/dist/chartjs-adapter-luxon.js
- on this part, it looks like new error arise, kindly help, see below error - The resource has been blocked

Failed to find a valid digest in the 'integrity' attribute for resource 'https://cdn.jsdelivr.net/npm/chartjs-adapter-luxon@1.0.0/dist/chartjs-adapter-luxon.js' with computed SHA-256 integrity 'tc58iTxh2t7CLhrhF1I4wFfgbuSM2UmT2HDlIHJSoyo='. The resource has been blocked.
chart.min.js:13 Uncaught Error: This method is not implemented: Check that a complete date adapter is provided.
    at no (chart.min.js:13)
    at oo.formats (chart.min.js:13)
    at Js.init (chart.min.js:13)
    at chart.min.js:13
    at J (chart.min.js:13)
    at to.buildOrUpdateScales (chart.min.js:13)
    at to.update (chart.min.js:13)
    at new to (chart.min.js:13)
    at mapYT2 (index.html:292)
    at HTMLDocument.<anonymous> (index.html:172)
no @ chart.min.js:13
formats @ chart.min.js:13
init @ chart.min.js:13
(anonymous) @ chart.min.js:13
J @ chart.min.js:13
buildOrUpdateScales @ chart.min.js:13
update @ chart.min.js:13
to @ chart.min.js:13
mapYT2 @ index.html:292
(anonymous) @ index.html:172

**Comment by AnderRV** [comment]
Yes, sorry, forgot to mention that when changing the URL, the integrity hash must me changed too.
The package is [here](https://www.jsdelivr.com/package/npm/chartjs-adapter-luxon), by the way.

```html
<script
  src="https://cdn.jsdelivr.net/npm/chartjs-adapter-luxon@1.1.0/dist/chartjs-adapter-luxon.js"
  integrity="sha256-q2NAytoP6eS3ONNWvLhccUNJ2kTlP++ZAhU89aIKt2Y="
  crossorigin="anonymous"
></script>
```

**Comment by edcarSky** [comment]
ok thanks, console no error at all, appreciated

but upon checking on zenrow perform request in the browser below is the error, any advise

Python:
                                            
import requests
response = requests.get("https://api.zenrows.com/v1/?apikey=4ef8379803d055ed0de9a73dc1947dfa8a905610&url=https://api.zenrows.com/v1/?apikey=4ef8379803d055ed0de9a73dc1947dfa8a905610&url=https://www.zenrows.com&autoparse=true")
print(response.json())

An error ocurred. Please try again later.
                                            
{
  "error": 1,
  "error_message": "could not automatically extract content. Please try Raw HTML endpoint"
}

**Comment by AnderRV** [comment]
Perfect! Good to hear that it went OK.

Now, as for the code error, autoparse will only work for some domains, such as YouTube here.

And you should not share your API key with anyone (not even us), so I suggest that you edit the comment above.

```
import requests
response = requests.get("https://api.zenrows.com/v1/?apikey=YOUR_API_KEY&url=https://www.youtube.com/watch?v=kE_VupU_8NQ&autoparse=true")
print(response.json())
```

Response

```
{'averageRating': 4.8578477, 'category': 'Music', 'description': '“She’s On Fire” the new music video by Billy F Gibbons from his new album ‘Hardware’. https://found.ee/BillyFGibbons-Hardware \n\nGet signed and exclusive bundles: https://found.ee/BillyFGibbons_Store \n\n \n\nDirected by:  Harry Reese\n\nProduced by:  Matt Sorum\n\n \n\nListen to ‘Hardware’: https://found.ee/BillyFGibbons-Hardware \n\niTunes https://found.ee/BillyFGibbons-Hardwa...\n\nApple Music https://found.ee/BillyFGibbons-Hardwa...\n\nSpotify https://found.ee/BillyFGibbons-Hardwa...\n\nAmazon Music https://found.ee/BillyFGibbons-Hardwa...\n\nAmazon https://found.ee/BillyFGibbons-Hardwa...\n\nYoutube Music https://found.ee/BillyFGibbons-Hardwa...\n\nPandora https://found.ee/BillyFGibbons-Hardwa...\n\nDeezer https://found.ee/BillyFGibbons-Hardwa...\n\n#BillyFGibbons #ShesOnFire #Hardware #OfficialMusicVideo\n\n\nTags: Billy Gibbons She’s on Fire, Billy F Gibbons She’s On Fire, She’s On Fire Billy F Gibbons, She’s On Fire Billy Gibbons, Billy Gibbons, Billy F Gibbons, Billy F Gibbons Hardware, She’s On Fire Official Music Video, She’s on Fire music video, She’s on Fire Hardware, Billy F Gibbons Hardware, Hardware Billy F Gibbons, Billy F Gibbons She’s On Fire Official Music Video, Billy Gibbons Hardware, music video, official video\n\nhttp://vevo.ly/1vJlqo', 'dislikes': 677, 'lengthSeconds': '167', 'likes': 18373, 'owner': 'Billy F Gibbons', 'publishDate': '2021-06-09', 'title': "Billy F Gibbons - She's On Fire", 'uploadDate': '2021-06-09', 'videoId': 'kE_VupU_8NQ', 'viewCount': 1460789}
```

**Comment by edcarSky** [comment]
thanks for the advise abt APIKEY, now the new issue

An error ocurred. Please try again later.

{
  "error": 1,
  "error_message": "could not automatically extract content. Please try Raw HTML endpoint"
}

**Comment by AnderRV** [comment]
"autoparse" will only work for some domains, zenrows.com is not one of them.
When `autoparse=true` we try to extract content automatically into a JSON object, but it is not activated for every domain. In those cases, we recommend removing that parameter y trying Raw HTML. Then, parse it (i.e., beautifulSoup in python) on your own.

**Comment by edcarSky** [comment]
{
  "error": 1,
  "error_message": "Malformed request"
}

**Comment by edcarSky** [comment]
ok now its running... what is next, how i will see the dashboard again, sorry for confusion

**Comment by AnderRV** [comment]
Sorry, but I think there's been a misunderstanding. We tracked videos that went viral and wrote how those evolve over time.

We cannot help you with the content or promo part on making videos, that's not our expertise.
